manifests/c/CorbinDavenport/NexusTools/5.7/CorbinDavenport.NexusTools.installer.yaml

```
@@ -0,0 +1,27 @@
+# Created with komac v2.11.2
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.9.0.schema.json
+
+PackageIdentifier: CorbinDavenport.NexusTools
+PackageVersion: '5.7'
+InstallerType: zip
+NestedInstallerType: exe
+NestedInstallerFiles:
+- RelativeFilePath: nexustools.exe
+InstallModes:
+- interactive
+- silent
+- silentWithProgress
+UpgradeBehavior: uninstallPrevious
+ReleaseDate: 2024-06-29
+InstallationMetadata:
+  DefaultInstallLocation: '%AppData%\NexusTools'
+InstallerSwitches:
+  Silent: nexustools -i -w
+  SilentWithProgress: nexustools -i -w
+  Repair: nexustools -r -w
```

```suggestion
  Repair: nexustools -r -w
  Dependencies:
    PackageDependencies:
    - PackageIdentifier: ClockworkMod.UniversalADBDriver
```

- See <https://github.com/microsoft/winget-pkgs/pull/246216#issuecomment-2785824087>